source/includes/auth.md

```diff
@@ -156,7 +275,26 @@ GET /openmrs/ws/rest/v1/location?tag=Login+Location'
 
 ```
 
-While fetching individual locations requires authentication, you can get a list of available locations by passing 
+```javascript
+
+var myHeaders = new Headers();
+myHeaders.append("Authorization", "Basic YWRtaW46QWRtaW4xMjM=");
+myHeaders.append("Cookie", "JSESSIONID=2D158E83ACFB788998C7DB495F07C1B9");
+
+var requestOptions = {
+  method: 'GET',
+  headers: myHeaders,
+  redirect: 'follow'
+};
+
+fetch("https://demo.openmrs.org/openmrs/ws/rest/v1/location?tag=Login+Location'\n", requestOptions)
+  .then(response => response.text())
```

Comment:
@Akayeshmantha  what is this API call supposed to return since I get a success response but it is empty.

Reply:
It should be the login location